api/spec/v1.yaml

```
@@ -1740,15 +1740,19 @@ components:
     RuntimeEvmBalance:
       description: Balance of an account for a specific runtime and EVM token.
       type: object
-      required: [balance, token_contract_addr, token_decimals, token_type]
+      required: [balance, token_contract_addr, token_contract_addr_eth, token_decimals, token_type]
       properties:
         balance:
           <<: *BigIntType
           description: Number of tokens held, in base units.
         token_contract_addr:
+          type: string
+          description: The oasis address of this token's contract.
+          example: 'oasis1qzk5pr2x8ah04lgjee3lv06fmyvqvz45egjv4ps0'
+        token_contract_addr_eth:
           type: string
           description: The EVM address of this token's contract. Encoded as a lowercase hex string.
```

```suggestion
          description: The EVM address of this token's contract. 
```

Thumbs up for proper stringification.